module/sidebar/components/menu-component.js

```diff
@@ -298,4 +308,144 @@ export default class MenuComponent extends Component {
             }
         });
     }
+
+    _prepareManeuvers(actor) {
+        this.lock = true
+        const maneuvers = new Array(
```

Comment:
new Array ain't necessary just make it

`[ data ]`

Reply:
I also do not want huge arrays of data in the middle of a file. Especially so when it's a component. Data like this should be in its own data file.

Besides, these moves are in the item compendium, instead of duplicating data, just retrieve them from the compendium